integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6579,6 +6584,14 @@ msgstr "Weitere Seiten-Infos"
 msgid "This organization currently has no maintained pages."
 msgstr "Von dieser Organisation werden zur Zeit keine Seiten verwaltet."
 
+#: cms/templates/organizations/organization_form.html
+msgid "View location"
+msgstr "Weitere Ort-Infos"
+
+#: cms/templates/organizations/organization_form.html
+msgid "This organization currently has no maintained locations."
+msgstr "Von dieser Organisation werden zur Zeit keine orte verwaltet."
```

Comment:
```suggestion
msgstr "Von dieser Organisation werden zur Zeit keine Orte verwaltet."
```
Nit: The "Tekamolo"-principle in German would suggest to put "Zur Zeit" at the beginning of the sentence as in "Zur Zeit werden von dieser Organisation keine Orte verwaltet". But your translation works just as well and also is commonly used :)

Reply:
This is inherited from the existing translation too. Should we once check all the translations for update/modification?

Reply:
Oh, I didn't realize this. Hm, on one hand it would be definitely helpful but I assume this will be a tedious task no one will ever do. So I'm unsure. Let's raise it in the next grooming :)